src/Navbar.jsx

```diff
@@ -50,23 +50,30 @@ const Navbar = ({ sessionToken, deleteSessionToken }) => {
         {sessionToken ? (
           <>
             <StyledListItem>
-              <StyledLink to={`/users/${sessionToken.userId}`}>
+              <StyledLink
+                className="profilePage"
+                to={`/users/${sessionToken.userId}`}
+              >
                 Profile
               </StyledLink>
             </StyledListItem>
             <StyledListItem>
-              <Button navLink onClick={logout}>
+              <Button classname="logout" navLink onClick={logout}>
```

Comment:
Don't forget this is React, so should be className

Reply:
Thanks that actually fixed an issue I had to find a workaround for in `Navbar.test`!